From: Simon McVittie <smcv@debian.org>
Date: Thu, 15 Feb 2018 01:08:23 +0000
Subject: build: Run install-test-data-hook as intended

This wasn't set up when it was copied from Flatpak.

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 Makefile.am | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Makefile.am b/Makefile.am
index b9b44d3..23bbd62 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -44,4 +44,7 @@ EXTRA_DIST += xdg-desktop-portal.pc.in
 
 EXTRA_DIST += README.md
 
+install-data-hook:
+	$(MAKE) $(AM_MAKEFLAGS) install-test-data-hook
+
 AM_DISTCHECK_CONFIGURE_FLAGS =
